movement/io/load_poses.py

```diff
@@ -274,10 +285,17 @@ def _load_from_sleap_labels_file(
     file = ValidHDF5(file_path, expected_datasets=["pred_points", "metadata"])
     labels = read_labels(file.path.as_posix())
     tracks_with_scores = _sleap_labels_to_numpy(labels)
+    individual_names = [track.name for track in labels.tracks] or None
+    if individual_names is None:
+        log_warning(
+            f"Could not find SLEAP Track in {file.path}. "
+            "Assuming single-individual dataset and assigning "
+            "default individual name."
```

Comment:
also consider mentionin what the default name is here

Reply:
i.e. "individual_0"

Reply:
I would say we don't, for maintainability, plus [`ValidPoseTracks`](https://github.com/neuroinformatics-unit/movement/blob/7e12f08b49dcab6ee3924861ad7ae6cfa3be172c/movement/io/validators.py#L346-L349) will tell what names are set

Reply:
I'm fine with that.